server/src/http/controllers/participant.controller.ts

```diff
@@ -61,4 +66,38 @@ export class ParticipantController {
   ): Promise<void> {
     await this.participantService.deleteParticipant(actor, participantId);
   }
+
+  @Get("/:participantId/records")
+  @ApiResponse({
+    status: 200,
+    description: "특정 참가자의 모든 기록 목록 반환",
+    type: [RecordResponseDto],
+  })
+  async getRecords(
+    @CurrentActor() actor: Actor,
+    @Param("participantId") participantId: string
+  ): Promise<RecordResponseDto[]> {
+    return this.recordService.getRecords(actor, participantId);
+  }
+
+  @Post("/:participantId/records")
```

Comment:
기록 하나를 추가하는 API이미 엔트포인트가 /records가 아닌 /record가 되는게 더 좋을 거 같네요
```suggestion
  @Post("/:participantId/record")
```

Reply:
복수 생성은 `/:participantId/records/batch` 형태로 RESTful한 API를 설계할 수 있다고 합니다.